jwql/website/apps/jwql/monitor_models/common.py

```diff
@@ -191,15 +191,15 @@ def db_for_read(self, model, **hints):
         """
```

Comment:
The `MonitorRouter` class should be in its own file called `router.py` and not associated with the common monitor functionality.  This router is a system wide addition (apologies, as this comment should have been addressed in the previous PR).  I personally think `router.py` should live parallel to `models.py` however if you choose to keep it in the sub folder thats fine.

Reply:
I decided to move it, and have implemented the name-based constant as described, and updated settings accordingly.